manifests/server.pp

```diff
@@ -0,0 +1,17 @@
+# ex: syntax=puppet si ts=4 sw=4 et
+
+define bind::server (
+    $bogus     = false,
+    $edns      = true,
+    $key       = undef,
+    $transfers = undef,
+    $order     = '10',
```

Comment:
I think you're correct in that the order doesn't matter.  I think i simply copied one of the other manifests and tweaked it.  Shouldn't be a problem to remove it.